integration_test/cases/migrator.exs

```diff
@@ -10,6 +10,8 @@ defmodule Ecto.Integration.MigratorTest do
   alias Ecto.Migration.SchemaMigration
 
   setup do
+    Ecto.Pools.Ownership.Server.ownership_checkout(TestRepo.Pool,
```

Comment:
own_worker? buy_worker? :)

@michalmuskala's name idea lends itself to better names here.